vidarr-server/src/main/resources/ca/on/oicr/gsi/vidarr/server/api-docs/vidarr.json

```diff
@@ -903,8 +1077,7 @@
                 "$ref": "#/components/schemas/CopyUnloadRequest"
               }
             }
-          },
-          "description": "Available filters are: [\"and\", \"not\", \"or\", \"vidarr-analysis-id\", \"vidarr-completed-after\", \"vidarr-external-id\", \"vidarr-external-provider\", \"vidarr-last-submitted-after\", \"vidarr-workflow-id\", \"vidarr-workflow-name\", \"vidarr-workflow-run-id\"]. Filters can be combined."
```

Comment:
With the update you've made, I expected to see the list of these filters in the "Example Value | Schema" portion of the deployed OpenAPI docs, but that list isn't there. If it's not intended to be there, could this description please be maintained to make the list accessible for users?

Reply:
It should appear on the Schema tab: https://stackoverflow.com/questions/73238665/how-to-display-all-of-the-oneof-schemas-in-swagger-ui

Reply:
Hmm, it looks like the Schema tab is showing an empty object for `filter`.

Reply:
Okay. I don't know how to debug the Swagger UI.

Reply:
Fair enough! I'd like to leave the description in so users have easier access to the possible values; would you like to modify the PR or would you like me to make the update?